features/bugsnag-errors-integration.feature

```diff
@@ -5,20 +5,30 @@ Feature: bugsnag errors integration
 Scenario: It picks up configuration from bugsnag errors' environment variables
   Given I set environment variable "BUGSNAG_API_KEY" to "ab123456789012345678901234567890"
   Given I set environment variable "BUGSNAG_RELEASE_STAGE" to "developroduction"
+  Given I set environment variable "OTEL_SERVICE_NAME" to "srv1"
```

Comment:
These are the tests for the integration with the errors SDK, so I think it only makes sense to add the app version check here and add this to the manual feature tests.

Reply:
added new tests